lib/open_api_spex/cast/array.ex

```diff
@@ -16,7 +17,43 @@ defmodule OpenApiSpex.Cast.Array do
 
   ## Private functions
 
-  defp cast_items(%{value: items} = ctx) do
+  defp cast_array(%{value: value, schema: %{minItems: minimum}} = ctx)
+       when is_list(value) and is_integer(minimum) do
+    item_count = Enum.count(value)
+
+    if item_count < minimum do
+      Cast.error(ctx, {:min_items, minimum, item_count})
+    else
+      {:ok, value}
+    end
+  end
+
+  defp cast_array(%{value: value, schema: %{maxItems: maximum}} = ctx)
+       when is_list(value) and is_integer(maximum) do
+    item_count = Enum.count(value)
+
+    if item_count > maximum do
+      Cast.error(ctx, {:max_items, maximum, item_count})
+    else
+      {:ok, value}
+    end
+  end
+
+  defp cast_array(%{value: value, schema: %{uniqueItems: true}} = ctx)
+       when is_list(value) do
+    unique_size =
```

Comment:
This seems a little more direct: `unique_size = value |> Enum.unique() |> length()`

Reply:
Ok, cool!